examples/nidigital_spi/measurement.py

```diff
@@ -35,6 +32,7 @@
 @measurement_service.configuration("levels_file_path", nims.DataType.Path, "PinLevels.digilevels")
 @measurement_service.configuration("timing_file_path", nims.DataType.Path, "Timing.digitiming")
 @measurement_service.configuration("pattern_file_path", nims.DataType.Path, "Pattern.digipat")
+@measurement_service.configuration("do_first_run_setup", nims.DataType.Boolean, True)
```

Comment:
I think it might make more sense to just call this `load_files` or something like that. Maybe putting a decorator around the group of controls that are impacted by the Boolean would also make sense? Ideally, the controls would disabled when the check box isn't checked, but we don't currently have those capabilities in the meas UI editor.

Reply:
I had it as 'load_files' originally which is accurate. But I wanted this to also serve as an example of how to do one-time setup for any measurement. I'd prefer to keep it as is since the comment explains its usage. I'm not sure what kind of decorator you're referring to.

Reply:
![image](https://github.com/ni/measurementlink-python/assets/103200169/95af3826-84af-4c74-a23f-ea292e74ed34)

I guess the palette is called "Drawings". They are called "Decorations" in LV. Basically, any control that is there for purely cosmetic reasons and can't be bound to data. In this case, the purpose would be to provide a visual cue that this other group of controls are related / coupled in some fashion. You could also use a canvas for this. 

I don't know that I agree that users will understand this as a conceptual example for how to do one time inititialization with measurements. If that is the goal of this example, we should probably rename the example with that concept in mind or have some other mechanism for users to map these concepts to examples which demonstrate the concept in action.

With that said, I don't feel strongly enough about any of this to require any action so I'll leave it up to you on what if anything you want to do.

Reply:
Oh I see. When you said 'decorator' I was thinking of Python decorators like '@measurement_service.configuration'. I'll take a look at that.